plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-login.spec.js

```diff
@@ -54,20 +63,10 @@ test.describe( 'Shopper Checkout Login Account', () => {
 		await api.post( `shipping/zones/${ shippingZoneId }/methods`, {
 			method_id: 'free_shipping',
 		} );
-		// update customer billing details.
-		await api.put( 'customers/2', {
-			billing: {
-				first_name,
-				last_name,
-				address_1,
-				address_2,
-				city,
-				state,
-				postcode,
-				country,
-				phone,
-			},
-		} );
+		// create customer and save its id
+		await api
+			.post( 'customers', customer )
+			.then( ( response ) => ( customerId = response.data.id ) );
```

Comment:
There is no customer with id = 2 in the nightly smoke test site, so I updated this test to create its own test customer.